fpdf/fpdf.py

```diff
@@ -1562,10 +1563,17 @@ def regular_polygon(self, x, y, numSides, polyWidth, rotateDegrees=0, style=None
         # center point is (centerX, centerY)
         points = []
         for i in range(1, numSides + 1):
-            point = centerX + radius * math.cos(
-                math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
-            ), centerY + radius * math.sin(
-                math.radians((360 / numSides) * i) + math.radians(rotateDegrees)
+            point = (
```

Comment:
Could you <ins>not</ins> incluse those changes please? They seem not needed to me a part of this PR